CONTRIBUTING.md

```diff
@@ -30,11 +30,8 @@ Following the [instructions in the README](README.md#setup) is enough.
     If you have a slow internet connection
     and it takes a long time for `windows_setup.sh` to download mingw64,
     you can instead run `./windows_setup.sh --small`.
-    This way it uses `mingw64-small.zip`,
-    which is just like the usual mingw64, but with many large files deleted to make it smaller.
-    I created it locally on my computer.
-    If you don't want to trust it, you can run `windows_setup.sh` without `--small`
-    or look at how `.github/workflows/windows.yml` compares `mingw64-small.zip` to the original `mingw64.zip`.
+    Instead of downloading the full mingw64 (about 1GB),
+    this will get a minimal version of MinGW from a [release](#releases) of Jou (about 50MB).
```

Comment:
```suggestion
    Instead of downloading the full mingw64 (about 360MB download),
    this will get a minimal version of MinGW from a [release](#releases) of Jou (about 50MB download).
```